spec/features/hiring_staff_can_sign_in_with_dfe_spec.rb

```diff
@@ -303,7 +303,7 @@
   end
 
   def sign_in_user
-    click_on(I18n.t('nav.sign_in'))
+    first(:link, I18n.t('nav.sign_in')).click
```

Comment:
Are there multiple links which match this now?

Reply:
Yep, the link is both in header and footer

Reply:
This is just a suggestion: since this method is defined the same in two different specs, could we define it once (in a (new?) helper file for our specs. The file would then contain definitions for methods that are defined & used repeatedly in our tests for setup)?